core/arch/arm/mm/core_mmu.c

```diff
@@ -118,91 +254,153 @@ static struct map_area *find_map_by_pa(unsigned long pa)
 	return NULL;
 }
 
-static void insert_mmap(struct tee_mmap_region *mm, size_t max_elem,
-		struct tee_mmap_region *mme)
+extern const struct core_mmu_phys_mem __start_phys_mem_map_section;
+extern const struct core_mmu_phys_mem __stop_phys_mem_map_section;
```

Comment:
Not super important, but I think you more often read "end" instead of "stop" when it comes to regions etc.